packages/portfolio-deploy/src/chain-info.core.js

```diff
@@ -118,18 +123,18 @@ const publishAxelarChainInfo = async (agoricNamesNode, axelarConfig) => {
  *
  * @param {BootstrapPowers & ChainStoragePresent} powers
  * @param {{
- *   options?: {
+ *   options: {
  *     chainInfo?: Record<string, ChainInfo>;
- *     axelarConfig?: import('./axelar-configs').AxelarChainConfigMap;
+ *     axelarConfig: import('./axelar-configs.js').AxelarChainConfigMap;
```

Comment:
I'd expect the axelarId to be part of chainInfo; the other option would be a map from chain name to contract addresses